app/models/order.rb

```diff
@@ -1,5 +1,6 @@
 class Order < ApplicationRecord
   has_many :orderitems
+  #is the line above needed? or redundant using through
   has_many :products, through: :order_items
```

Comment:
I THINK we need both line 2 and 3. I do have a q about the orderitems vs order_items syntax. Maybe we can check in rails console if one or the other or both works?